functions/Set-DbaErrorLog.ps1

```diff
@@ -35,130 +35,98 @@ function Set-DbaErrorLog {
 
         .EXAMPLE
-            Set-DbaErrorLog -SqlInstance server2012 -NumberOfLog 25 -SizeInKb 500
+            Set-DbaErrorLog -SqlInstance sql2012 -LogCount 25 -LogSize 500
 
-            Sets the number of error log files to 25 and size before it will roll over to 500KB on server2012
+            Sets the number of error log files to 25 and size before it will roll over to 500 KB on sql2012
     #>
     [cmdletbinding(SupportsShouldProcess)]
     param(
-        [Parameter(ValueFromPipelineByPropertyName)]
+        [Parameter(ValueFromPipelineByPropertyName, Mandatory)]
         [DbaInstanceParameter[]]$SqlInstance,
         [PSCredential]$SqlCredential,
-        [DbaValidatePattern('^[6-9][0-9]?$', ErrorMessage = "Error processing {0} - input must be an integer between 6 and 99")]
-        [int]$NumberOfLog,
-        [int]$SizeInKb,
+        [ValidateRange(6, 99)]
+        [int]$LogCount,
+        [int]$LogSize,
         [switch]$EnableException
     )
     process {
         foreach ($instance in $SqlInstance) {
             Write-Message -Level Verbose -Message "Connecting to $instance"
             try {
-                $server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $SqlCredential
+                $server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $SqlCredential -MinimumVersion 11
             }
             catch {
                 Stop-Function -Message "Failure" -Category ConnectionError -ErrorRecord $_ -Target $instance -Continue
             }
-
-            $collectionNumberLog = [PSCustomObject]@{
-                ComputerName            = $server.NetName
-                InstanceName            = $server.ServiceName
-                SqlInstance             = $server.DomainInstanceName
-                Setting                 = 'NumberOfLogFiles'
-                OriginalNumberErrorLogs = $null
-                CurrentNumberErrorLogs  = $null
-                Status                  = $null
-            }
-            $collectionErrorLogSize = [PSCustomObject]@{
-                ComputerName           = $server.NetName
-                InstanceName           = $server.ServiceName
-                SqlInstance            = $server.DomainInstanceName
-                Setting                = 'ErrorLogSizeKb'
-                OriginalErrorLogSizeKb = $null
-                CurrentErrorLogSizeKb  = $null
-                Status                 = $null
+            
+            $currentNumLogs = $server.NumberOfLogFiles
+            $currentLogSize = $server.ErrorLogSizeKb
+            
+            $collection = [PSCustomObject]@{
+                ComputerName              = $server.NetName
+                InstanceName              = $server.ServiceName
+                SqlInstance               = $server.DomainInstanceName
+                LogCount                  = $currentNumLogs
+                LogSize                   = [dbasize]($currentLogSize * 1024)
```

Comment:
This change would also need to be done to the Get-DbaErrorLogConfig so the output matches.